### ISSUE TITLE
Split Panes don't work / sometimes raise errors

### ISSUE THREAD

**Issue by vlouppe** [bigfive/atom-sublime-select#34]
This is how you can reproduce the bug :

1. open 2 files
2. split the screen with (View/Panes/Split Right)
3. try to select a column with ALT + Click, nothing happens

**Atom Version**: 0.175.0
**System**: Mac OS X 10.10.1
**Thrown From**: [Sublime-Style-Column-Selection](https://github.com/bigfive/atom-sublime-select) package, v1.2.2

### Stack Trace

Uncaught TypeError: Cannot read property 'getDOMNode' of undefined

```
At /Applications/Atom.app/Contents/Resources/app/src/text-editor-component.js:1290

TypeError: Cannot read property 'getDOMNode' of undefined
  at [object Object].module.exports.React.createClass.pixelPositionForMouseEvent (/Applications/Atom.app/Contents/Resources/app/src/text-editor-component.js:1290:40)
  at [object Object].boundMethod [as pixelPositionForMouseEvent] (/Applications/Atom.app/Contents/Resources/app/node_modules/react-atom-fork/lib/ReactCompositeComponent.js:1295:21)
  at /Users/vincent/.atom/packages/Sublime-Style-Column-Selection/lib/sublime-select.coffee:72:42
  at atom-text-editor.<anonymous> (/Users/vincent/.atom/packages/Sublime-Style-Column-Selection/lib/sublime-select.coffee:45:23)

```

### Commands

```
     -0:00.0 pane:split-right (atom-text-editor.editor)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
Sublime-Style-Column-Selection, v1.2.2

# Dev
No dev packages
```

/cc @atom/core

**Comment by karai17** [comment]
1. Middle-click to vertically select text
2. broke!

**Atom Version**: 0.177.0
**System**: Microsoft Windows 7 Professional 
**Thrown From**: [Sublime-Style-Column-Selection](https://github.com/bigfive/atom-sublime-select) package, v1.2.3

### Stack Trace

Uncaught TypeError: Cannot read property 'getDOMNode' of undefined

```
At C:\Users\Karai\AppData\Local\atom\app-0.177.0\resources\app\src\text-editor-component.js:1290

TypeError: Cannot read property 'getDOMNode' of undefined
  at [object Object].module.exports.React.createClass.pixelPositionForMouseEvent (C:\Users\Karai\AppData\Local\atom\app-0.177.0\resources\app\src\text-editor-component.js:1290:40)
  at [object Object].boundMethod [as pixelPositionForMouseEvent] (C:\Users\Karai\AppData\Local\atom\app-0.177.0\resources\app\node_modules\react-atom-fork\lib\ReactCompositeComponent.js:1295:21)
  at C:\Users\Karai\.atom\packages\Sublime-Style-Column-Selection\lib\sublime-select.coffee:76:42
  at atom-text-editor.<anonymous> (C:\Users\Karai\.atom\packages\Sublime-Style-Column-Selection\lib\sublime-select.coffee:49:23)

```

### Commands

```
     -9:26.9 editor:newline (atom-text-editor.editor)
     -9:20.7 core:move-right (atom-text-editor.editor)
     -9:18.6 core:move-up (atom-text-editor.editor)
  3x -9:18.5 core:move-right (atom-text-editor.editor)
     -9:17.0 core:save (atom-text-editor.editor)
     -9:00.8 editor:newline (atom-text-editor.editor)
     -8:59.8 snippets:next-tab-stop (atom-text-editor.editor)
     -8:59.8 snippets:expand (atom-text-editor.editor)
     -8:59.8 editor:indent (atom-text-editor.editor)
     -8:51.5 core:save (atom-text-editor.editor)
     -8:33.1 snippets:next-tab-stop (atom-text-editor.editor)
     -8:33.1 snippets:expand (atom-text-editor.editor)
     -8:33.1 editor:indent (atom-text-editor.editor)
  2x -8:10.9 build:trigger (atom-text-editor.editor)
  3x -0:19.8 find-and-replace:select-next (atom-text-editor.editor)
  2x -0:18.5 core:delete (atom-text-editor.editor)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "deprecation-cop",
      "language-objective-c",
      "language-clojure",
      "language-coffee-script",
      "language-go",
      "language-java",
      "language-less",
      "language-mustache",
      "language-perl",
      "language-php",
      "language-python",
      "language-ruby",
      "language-ruby-on-rails",
      "language-sass",
      "language-source",
      "language-toml",
      "language-yaml",
      "welcome",
      "wrap-guide",
      "fuzzy-finder",
      "column-select"
    ],
    "themes": [
      "atom-dark-ui",
      "monokai"
    ]
  }
}
```

### Installed Packages

```coffee
# User
build, v0.23.3
color-picker, v1.4.4
language-lua, v0.9.0
minimap, v4.1.0
monokai, v0.12.0
motepair, v0.7.0
project-manager, v1.15.5
slickedit-select, v1.3.0
Sublime-Style-Column-Selection, v1.2.3

# Dev
No dev packages
```

/cc @atom/core

**Comment by bigfive** [comment]
This should now be fixed in the latest release (v1.4.1)
